app/request_lookup.py

```diff
@@ -46,26 +78,69 @@ def get_request_json(data_type, search_value):
         }
     ]
 
+    if language != 'en':
+        should_terms.append({
```

Comment:
Not worked with ES before, is the order they are appended important or is it just based on the `boost` value?

Reply:
Order isn't important, it orders them by the score